.github/workflows/pylint.yml

```diff
@@ -11,31 +11,33 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        python-version: [ 3.9, '3.10']
```

Comment:
We should keep all supported python version, because may act differently in different setups. Or what was your reason?

Reply:
This is the linting, it works the same across different python versions